datahub-frontend/conf/application.conf

```diff
@@ -203,6 +203,11 @@ auth.oidc.grantType = ${?AUTH_OIDC_GRANT_TYPE}
 #
 auth.jaas.enabled = ${?AUTH_JAAS_ENABLED}
 auth.native.enabled = ${?AUTH_NATIVE_ENABLED}
+auth.guest.enabled = ${?GUEST_AUTHENTICATION_ENABLED}
```

Comment:
Not a big deal however the spring configuration is available from the ConfigurationProvider which sources these from `application.yml`. The use of this application.conf pre-dates the Spring configuration stuff. This works so not a blocker.